README.md

```diff
@@ -4,65 +4,40 @@
 <b>Electrode</b>
 </p>
 
-[![Gitter](https://badges.gitter.im/gitterHQ/gitter.svg)](https://gitter.im/electrode-io/electrode)
-[![Build Status][travis-image]][travis-url] [![devDependencies Status][daviddm-image]][daviddm-url]
+# Building large scale universal React Web Applications
 
-### Building large scale universal React Web Applications
+Electrode Web is a rapid application development framework; a full stack, end-to-end platform for developing, deploying, and maintaining JavaScript applications at Walmart, afor the <http://www.walmart.com> eCommerce website.
 
-> Built at @WalmartLabs powering the <http://www.walmart.com> eCommerce site, Electrode is a platform for building small to large scale universal ReactJS/NodeJS web applications with a standardized structure, best practices, and modern technologies baked in. Electrode focuses on performance, component reusability, and simple deployment to multiple cloud providers—so you can focus on what makes your app unique.
+We maintain additional code to support internal systems.
```

Comment:
For Walmart applications, we maintain additional code to support proprietary internal systems.  If you are a Walmart developer working on internal applications, please check internal docs to get started.